### PR TITLE
Add load testing scripts, tweak some files to accomodate

### PR DESCRIPTION
**Pull request by cywang117**
### Features
- Add k6 load test scripts (Rename some require calls in `db/data-gen` to include '.js' for k6's benefit)
- Refactor to multi-cluster server to improve performance at scale
- Add k6 scripts in package.json

### Bugfixes
- Change single quotes around query strings in `db/models/mariaDB_descriptions.js` to double quotes to prevent a rare error where insert strings include apostrophes

__Ignore__: legacy Artillery yaml in `server/load-testing/`

@RaneWallin @margotTenenbaum @krisdo Not much to review here, honestly. The only thing that's worth looking at is the scripts in `server/load-testing`. Can't really write new test suites over this content either. 

Proxy PR over the same content is coming up (probably in a few days). Ty ty!

### CODE REVIEW

**Review by RaneWallin — APPROVED** [review]
Looks good. k5 looks interesting. I'm going to check it out.

File: server/load-testing/LEGACY-artillery-GET-genres.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+config:
+  target: "http://localhost:3003"
+  phases:
+    # Since 1, 10, or 100 RPS do not pose a problem to the server currently, they won't be tested for a
```
Comment (margotTenenbaum) [inline server/load-testing/LEGACY-artillery-GET-genres.yml:4]: Helpful comments! Seems like a great strategy for testing the most useful #'s of rps.

File: server/load-testing/LEGACY-generateProductIds.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  // Declare listeners
+  stream.on('error', err => {
+    throw err;
+  });
+  stream.on('close', () => {
+    console.log(`Finished writing ids to CSV. Check ${writePath}`);
+  });
+
+  // Write nums 9e6 - 1e7 to CSV
+  for (let i = 9e6; i <= 1e7; i++) {
```
Comment (margotTenenbaum) [inline server/load-testing/LEGACY-generateProductIds.js:20]: Efficient way to generate your product_id csv!

**Review by margotTenenbaum — COMMENTED** [review]
Approved! (just noticed Rane already commented) Liked your use of k6 as well.

**Pull request merged**